apps/src/templates/rubrics/LearningGoals.jsx

```diff
@@ -442,18 +458,21 @@ export default function LearningGoals({
 
     // Annotate the lines based on the AI observation
     clearAnnotations();
-    const aiEvalInfo = getAiInfo(learningGoals[currentIndex].id);
-    if (!!aiEvalInfo && aiEvalInfo.evidence) {
-      setAiEvidence(annotateLines(aiEvalInfo.evidence));
-    }
+    if (currentIndex !== learningGoals.length) {
+      const aiEvalInfo = getAiInfo(learningGoals[currentIndex].id);
+      if (!!aiEvalInfo && aiEvalInfo.evidence) {
+        setAiEvidence(annotateLines(aiEvalInfo.evidence));
+      }
 
-    if (!isStudent) {
-      const eventName = EVENTS.TA_RUBRIC_LEARNING_GOAL_SELECTED;
-      analyticsReporter.sendEvent(eventName, {
-        ...(reportingData || {}),
-        learningGoalKey: learningGoals[currentIndex].key,
-        learningGoal: learningGoals[currentIndex].learningGoal,
-      });
+      if (!isStudent) {
+        console.group(`Send Event`);
```

Comment:
Do you need to ungroup at the end of this event?

Reply:
oops I think this is left over from debugging, lemme remove it